src/network/modules/ledger/ledger.ts

```diff
@@ -10,8 +10,6 @@ interface Ledger extends NetworkModule {
   _namespace_: string
   info: () => Promise<LedgerInfo>
   balance: (address?: string, symbols?: string[]) => Promise<Balances>
-  mint: () => Promise<unknown>
-  burn: () => Promise<unknown>
```

Comment:
`mint` and `burn` are no longer in the `ledger` namespace (for a long time now).